### DISCUSSION TITLE
Installing package from rpm.pbone.net

### DISCUSSION THREAD

**Discussion by CarrotManMatt** [blue-build/.github#15] [Support]
I have found this package that I would like to add to my built image: `libpam-pwdfile`

The link to the repository location is here: https://rpm.pbone.net/info_idpl_54053641_distro_opensuseleap42_com_pam_pwdfile-1.0-3.3.x86_64.rpm.html

What would be the correct URL to put within the `repos` list of my `rpm-ostree` module to install `libpam-pwdfile`

**Comment by xynydev** [comment]
I'm having trouble parsing the webpage you have linked. You can add the link to the `repos:` **if it is a `.repo` file**. If you have a direct link to an `.rpm` file, you should be able to add it to the `install:` list directly.

If you're unable to find either one of those, I'm not sure what to do tbh. Finding some installation instructions somewhere would be useful.

**Reply by CarrotManMatt** [reply]
there seems to be a `ftp://` link, would `install: ` support that?

**Reply by xynydev** [reply]
Try it! If not, you can use a script and [DL with cURL](https://superuser.com/questions/265062/downloading-file-from-ftp-using-curl).
